packages/build/test-d/options/netlify_plugin_functions_util.ts

```diff
@@ -0,0 +1,35 @@
+import { OnPreBuild, NetlifyPluginUtils, ListedFunction, ListedFunctionFile } from '@netlify/build'
+import { expectType } from 'tsd'
+
+const testUtilsFunctionsList: OnPreBuild = function ({
+  utils: {
+    functions: { list },
+  },
+}: {
+  utils: NetlifyPluginUtils
```

Comment:
I believe `: { utils: NetlifyPluginUtils }` might not be needed here (or in the two functions below) because `tsc` infers it from the `OnPreBuild` type. I might be wrong though.